src/pages/homeCollection/kitsReceipt.js

```diff
@@ -0,0 +1,232 @@
+        obj[conceptIds.receivedDateTime] = storeDateReceivedinISO(document.getElementById('dateReceived').value);
+        if(document.getElementById('collectionId').value) {
+          obj[conceptIds.collectionCupId] = document.getElementById('collectionId').value;
+          const dateCollectionCard = document.getElementById('dateCollectionCard').value;
+          const timeCollectionCard = document.getElementById('timeCollectionCard').value;
+          obj[conceptIds.collectionDateTimeStamp] = dateCollectionCard + 'T' + timeCollectionCard
+          document.getElementById('collectionCheckBox').checked === true ? 
+              obj[conceptIds.collectionCardFlag] = true : obj[conceptIds.collectionCardFlag] = false
+          obj[conceptIds.collectionAddtnlNotes] = document.getElementById('collectionComments').value;
+          }    
+        
+        window.removeEventListener("beforeunload",beforeUnloadMessage)
+        targetAnchorTagEl();
+        storePackageReceipt(obj);
+       } 
+    })
+  }
+
+
+}
+
+const identifyCourierType = (scannedBarcode) => {
+  if (scannedBarcode.length === 20 || scannedBarcode.length === 22) {
+     return true
+   }
+   else {
+     return false
+ }}
+
+ const storePackageReceipt = async (data) => {
+  showAnimation();
+  const idToken = await getIdToken();
+  const response = await fetch(`${baseAPI}api=kitReceipt`,
+      {
+          method: "POST",
+          body: JSON.stringify(data),
+          headers: {
+              Authorization: "Bearer " + idToken,
+              "Content-Type": "application/json",
+          },
+      }
+  );
+  hideAnimation();
+  
+  if (response.status === 200) {
+    triggerSuccessModal('Kit Receipted.')
+    document.getElementById("courierType").innerHTML = ``;
+    document.getElementById("scannedBarcode").value = "";
+    document.getElementById("packageCondition").value = "";
+    document.getElementById("receivePackageComments").value = "";
+    document.getElementById("dateReceived").value = getCurrentDate();
+    document.getElementById("collectionComments").value = "";
+    document.getElementById("collectionId").value = "";
+    enableCollectionCardFields()
+    enableCollectionCheckBox()
+    document.getElementById("packageCondition").setAttribute("data-selected","[]")
+    if (document.getElementById("collectionId").value) {
+      document.getElementById("collectionId").value = "";
+      document.getElementById("dateCollectionCard").value = "";
+      document.getElementById("timeCollectionCard").value = "";
+      document.getElementById("collectionCheckBox").checked = false;
+      document.getElementById("collectionComments").value = "";
+
+      enableCollectionCardFields();
+      enableCollectionCheckBox();
+      document.getElementById("packageCondition").setAttribute("data-selected","[]");
+
+    }
+} else {
+  triggerSuccessModal('Kit Receipted.')
+      alert("Error");
```

Comment:
`alert("Error")` may not be needed since the error modal displays a more detailed message.